draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -192,122 +193,144 @@ For example, two quota policies containing further details via extension paramet
+When the value of the remaining keyword is low, it indicates that the server may soon throttle the client (see {{providing-ratelimit-fields}}).
 
-These examples show multiple policies being returned:
+For example:
 
 ~~~ example
-   RateLimit-Policy: 10;w=1, 50;w=60, 1000;w=3600, 5000;w=86400
-   RateLimit-Policy: 10;w=1;burst=1000, 1000;w=3600
+   remaining=50
 ~~~
 
-This field cannot appear in a trailer section.
 
-## RateLimit-Remaining {#ratelimit-remaining-field}
+## Reset Keyword {#ratelimit-reset-keyword}
 
-The "RateLimit-Remaining" response header field indicates the remaining quota units associated with the expiring-limit.
+The "reset" keyword indicates the number of seconds until the quota associated with the expiring-limit resets.
 
-The field is an Item and its value is a non-negative Integer expressed in [quota units](#service-limit).
-This specification does not define Parameters for this field.
-If they appear, they MUST be ignored.
+It is a non-negative Integer compatible with the delay-seconds rule, because:
 
-Clients MUST NOT assume that a positive RateLimit-Remaining header field value is a guarantee that further requests will be served.
+- it does not rely on clock synchronization and is resilient to clock adjustment
+  and clock skew between client and server (see {{Section 5.6.7 of HTTP}});
+- it mitigates the risk related to thundering herd when too many clients are serviced with the same timestamp.
 
-When the value of RateLimit-Remaining is low, it indicates that the server may soon throttle the client (see {{providing-ratelimit-fields}}).
+This specification does not define Parameters for it.
+If they appear, they MUST be ignored.
 
 For example:
 
 ~~~ example
-   RateLimit-Remaining: 50
+   reset=50
 ~~~
 
-This field cannot appear in a trailer section.
+The client MUST NOT assume that all its service limit will be reset at the moment indicated by the reset keyword. The server MAY arbitrarily alter the reset keyword value between subsequent requests; for example, in case of resource saturation or to implement sliding window policies.
 
-## RateLimit-Reset {#ratelimit-reset-field}
 
-The "RateLimit-Reset" response header field indicates the number of seconds until the quota associated with the expiring-limit resets.
+## RateLimit-Policy {#ratelimit-policy-field}
 
-The field is a non-negative Integer compatible with the delay-seconds rule, because:
+The "RateLimit-Policy" response header field indicates the quota policies currently associated with the client. Its value is informative.
 
-- it does not rely on clock synchronization and is resilient to clock adjustment
-  and clock skew between client and server (see {{Section 5.6.7 of HTTP}});
-- it mitigates the risk related to thundering herd when too many clients are serviced with the same timestamp.
+The field is a non-empty List of Items. Each item is a [quota policy](#quota-policy).
+Two quota policies MUST NOT be associated with the same quota units value.
```

Comment:
What is the reason for adding this constraint? And doesn't it contradict the example shown below with multiple time windows?
````
RateLimit-Policy: 10;w=1, 50;w=60, 1000;w=3600, 5000;w=86400
````
Is your assertion that each of these policies has a different quota unit?

Reply:
> Two quota policies MUST NOT be associated with the same quota units value.

I meant that the Policy field below is invalid

```
Policy: 10;w=1, 10;w=10
```